lib/ruby_aem/resources/config_property.rb

```diff
@@ -29,24 +29,20 @@ def initialize(client, name, type, value)
         @call_params = {
           name: name,
           type: type,
-          value: value
+          value: value,
+          query_params: { }
         }
+        @call_params[:query_params][@call_params[:name]] = @call_params[:value]
```

Comment:
@cliffano  do you have any opinion about this line ? I couldn't find a better way to solve this than setting the `query_params` for the API call directly in ruby_aem.